tests/test_compose.py

```diff
@@ -0,0 +1,106 @@
+import unittest
+
+import numpy as np
+import torch
+from numpy.testing import assert_almost_equal, assert_array_equal
+
+from torch_audiomentations import PolarityInversion, Compose, PeakNormalization
+from torch_audiomentations.augmentations.gain import Gain
```

Comment:
Why isn't it imported from `torch_audiomentations` like the other ones?

Reply:
Oh, that's probably an error. Let me fix it.